src/associations/helpers.ts

```diff
@@ -217,7 +271,11 @@ export function defineAssociation<
   construct: (opts: CleanOptions) => T,
 ): T {
   if (!isModelStatic(target)) {
-    throw new Error(`${source.name}.${lowerFirst(type.name)} called with something that's not a subclass of Sequelize.Model`);
+    throw new Error(
+      `${source.name}.${lowerFirst(
+        type.name,
+      )} called with ${target} that's not a subclass of Sequelize.Model`,
```

Comment:
There's a lot of styling changes in this PR. Is it true that this is the only actual change?

Reply:
Yes, this was the only change. Sorry! The rest was auto formatting that happened on commit.